src/serena/agent.py

```diff
@@ -37,6 +37,67 @@
 SUCCESS_RESULT = "OK"
 
 
+class ProjectConfig(ToStringMixin):
+    SERENA_MANAGED_DIR = ".serena"
+
+    def __init__(self, config_dict: dict[str, Any], project_name: str):
+        self.project_name: str = project_name
+        self.language: Language = Language(config_dict["language"])
+        self.project_root: str = str(Path(config_dict["project_root"]).resolve())
+        self.ignored_dirs: list[str] = config_dict.get("ignored_dirs", [])
+        self.excluded_tools: set[str] = set(config_dict.get("excluded_tools", []))
+
+    def _tostring_excludes(self) -> list[str]:
```

Comment:
why that? It's rather relevant info, I'd say

Reply:
Yeah, you're probably right. I'll remove it.